closure/testing/test/BUILD

```diff
@@ -111,6 +111,46 @@ closure_js_test(
     ],
 )
 
+# Reproduce issue with closure_js_library targets included in exports not
+# showing up as data in a test.
+
+closure_js_library(
+    name = "external",
+    srcs = ["external.js"],
+)
+
+closure_js_library(
+    name = "external_externs",
+    srcs = ["external_externs.js"],
+)
+
+closure_js_library(
+    name = "all_external",
+    exports = [":external"],
+)
+
+closure_js_test(
+    name = "external_code_test",
+    srcs = ["external_code_test.js"],
+    html = "external_code_test.html",
+    entry_points = [
+        "rules_closure.ExternalCodeTest",
+    ],
+    data = [
+        # including external.js through a rollup target via exports should work.
+        ":all_external",
```

Comment:
Does it work if you add `:external`?

I'm not very familiar with how the data attribute works. Does it consider `files`, `runfiles`, or both as input? If `runfiles` are used, then it should be easy to add. If it uses `files`, then it also shouldn't work if `:external` is a direct dependency, and this is working as intended.

https://github.com/bazelbuild/rules_closure/blob/308b05b2419edb5c8ee0471b67a40403df940149/closure/compiler/closure_js_library.bzl#L428

Reply:
Yes, it works if I reference the library directly. I'm afraid I don't know how it works either.

Reply:
Okay, then, AFAICT, this is indeed a bug.
You should be able to fix it by adding `library.exports` to [this list](https://github.com/bazelbuild/rules_closure/blob/308b05b2419edb5c8ee0471b67a40403df940149/closure/compiler/closure_js_library.bzl#L432) :)

Reply:
Nailed it! That fixed the test. I've updated this pull request.